data_rentgen/server/errors/schemas/not_found.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+import http
+from typing import Any
+
+from pydantic import BaseModel
+from typing_extensions import Literal
+
+from data_rentgen.exceptions.entity import EntityNotFoundError
+from data_rentgen.server.errors.base import BaseErrorSchema
+from data_rentgen.server.errors.registration import register_error_response
+
+
+class NotFoundDetailsSchema(BaseModel):
+    entity_type: str
+    field: str
+    value: Any
+
+
+@register_error_response(
+    exception=EntityNotFoundError,
+    status=http.HTTPStatus.NOT_FOUND,
+)
+class NotFoundSchema(BaseErrorSchema):
+    code: Literal["not_found"] = "not_found"
+    details: NotFoundDetailsSchema
+
+    def to_exception(self) -> EntityNotFoundError:
```

Comment:
Actually, this is used only in Horizon client, to convert HTTP response to local exception object. This can be removed.